beater/beater.go

```diff
@@ -108,9 +108,7 @@ func New(b *beat.Beat, ucfg *common.Config) (beat.Beater, error) {
 		logger:  logger,
 	}
 
-	// setup pipelines if explicitly directed to or setup --pipelines and config is not set at all
-	shouldSetupPipelines := beaterConfig.Register.Ingest.Pipeline.isEnabled() ||
-		(b.InSetupCmd && beaterConfig.Register.Ingest.Pipeline.Enabled == nil)
+	shouldSetupPipelines := beaterConfig.Register.Ingest.Pipeline.isEnabled() || b.InSetupCmd
```

Comment:
With this change pipeline would always be setup when in `setup` cmd, ignoring the actual setting of `register.ingest.pipeline.enabled`.

Reply:
yes, is it wrong?

Reply:
From my understanding yes. The `setup` command should respect the configuration file, and when `register.ingest.pipeline.enabled` is set to `false`, then why should they be set up?

Reply:
i believe the fix is like e9c50c240b913977e244fd8eefebc5bf40430f31 then